intro.tex

```diff
@@ -1,9 +1,11 @@
 \section{Introduction}
 
-Rubin Observatory will be substantially completed by October 2025. In practice this means the Operations team \cite[see][]{RDO-018} will assume responsibility of the Observatory and regular operations: running the facility on Cerro Pach\'{o}n each night, transferring data over the long haul network to the USDF, processing and arching the data, etc. 
+By the end of October 2025 the Rubin Observatory will be substantially completed having passed its Construction Completeness Review \#3 (CCR3) and Opperations Readiness Review \#2.  This will signify the formal handover of activities from the construction project to Rubin Operations. In practice, this means the Operations team \cite[see][]{RDO-018} will assume day-to-day responsibility of the Observatory and its regular operations: running the facility on Cerro Pach\'{o}n each night, transferring data over the long haul network to the US Data Facility (USDF), processing and arching the data, and delivering data products to the user community via realtime alerts and scheduled data releases. 
```

Comment:
```suggestion
By the end of October 2025 the Rubin Observatory will be substantially completed having passed its Construction Completeness Review \#3 (CCR3) and Operations Readiness Review \#2.  This will signify the formal handover of activities from the construction project to Rubin Operations. In practice, this means the Operations team \cite[see][]{RDO-018} will assume day-to-day responsibility of the Observatory and its regular operations: running the facility on Cerro Pach\'{o}n each night, transferring data over the long haul network to the US Data Facility (USDF), processing and archiving the data, and delivering data products to the user community via realtime alerts and scheduled data releases. 
```